tools/tests/mentorship_type_test.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from automation import get_mentorship_type
+
+class TestGetMentorshipType(unittest.TestCase):
+    AD_HOC_1 = "Ad-Hoc Format"
+    AD_HOC_2 = "Ad Hoc Format"
+    LONG_TERM_1 = "Long-term Format"
+    LONG_TERM_2 = "Long term Format"
+    BOTH = "Both"
+    WRONG_FORMAT = "Wrong Format"
+    EMPTY = ""
+
+    def test_get_ad_hoc_type(self):
+        self.assertEqual("ad-hoc", get_mentorship_type(self.AD_HOC_1))
```

Comment:
Nit: 

you could export "ad-hoc" as variable from your script and re-use here, because in case we decided to change this later to be short-session instead of ad-hoc, you only need to change it would be simpler to do. As you created here the other variables to re-use.